grafonnet-base/main.libsonnet

```diff
@@ -187,9 +194,14 @@ local veneer = import './veneer/main.libsonnet';
   coreLib: {
     new(schema):
       local title = schema.info.title;
+      local spec =
+        if 'spec' in schema.components.schemas[title].properties
+        then schema.components.schemas[title].properties.spec
+        else schema.components.schemas[title];
```

Comment:
The new schemas render the core kinds as Kubernetes-like resources, this little check ensures we use the spec of those resources.